API/parseClassToStructs.m

```diff
@@ -210,10 +210,8 @@
             priorType = 1;
         elseif strcmpi(thisType,'gaussian')
             priorType = 2;
-        elseif strcmpi(thisType,'jeffreys')
-            priorType = 3;
         else
-            priorType = 1;
+            priorType = 3;
```

Comment:
The original code implied that there may be a case other than "uniform", "gaussian", "jeffreys". I take it this isn't the case?